diff --git a/profiles/work-common.nix b/profiles/work-common.nix
index d7f175d..1826950 100644
--- a/profiles/work-common.nix
+++ b/profiles/work-common.nix
@@ -26,13 +26,13 @@
   home = {
     file = {
       "jdks/openjdk8".source = pkgs.openjdk8;
-      "jdks/oraclejdk8".source = pkgs.callPackage (import ../pkgs/oraclejdk-linux-base.nix {
-        productVersion = "8";
-        patchVersion = "202";
-        jceName = "jce_policy-8.zip";
-        sha256JCE = "19n5wadargg3v8x76r7ayag6p2xz1bwhrgdzjs9f4i6fvxz9jr4w";
-        sha256.x86_64-linux = "1q4l8pymjvsvxfwaw0rdcnhryh1la2bvg5f4d4my41ka390k4p4s";
-      }) { };
+      # "jdks/oraclejdk8".source = pkgs.callPackage (import ../pkgs/oraclejdk-linux-base.nix {
+      #   productVersion = "8";
+      #   patchVersion = "202";
+      #   jceName = "jce_policy-8.zip";
+      #   sha256JCE = "19n5wadargg3v8x76r7ayag6p2xz1bwhrgdzjs9f4i6fvxz9jr4w";
+      #   sha256.x86_64-linux = "1q4l8pymjvsvxfwaw0rdcnhryh1la2bvg5f4d4my41ka390k4p4s";
+      # }) { };
       "jdks/openjdk11".source = pkgs.openjdk11;
       "jdks/openjdk17".source = pkgs.openjdk17;
       "jdks/openjdk21".source = pkgs.openjdk21;
@@ -81,9 +81,7 @@
       minio-client
       myPkgs.hunspell-lt
       keystore-explorer
-      (pkgs.callPackage ../pkgs/sqldeveloper {
-        jdk = pkgs.openjdk17.override { enableJavaFX = true; };
-      })
+      (pkgs.openjdk17.override { enableJavaFX = true; })
       mercurial
       pnpm_10
       podman-compose
